codes/utils.py

```diff
@@ -17,6 +17,8 @@
 adm3_dir_fmt = 'gadm36_{iso3}_{datestamp}.zip'
 
 CUM_CASE_MIN_FILTER = 10
+PROCESSED_DATA_ERROR_HANDLING = "warn"
+PROCESSED_DATA_DATE_CUTOFF = True
```

Comment:
```suggestion
PROCESSED_DATA_DATE_CUTOFF = False
```
since we're currently not cutting off processed data.